src/driver.rs

```diff
@@ -47,7 +47,7 @@ pub fn main() {
         if orig_args.len() <= 1 {
             std::process::exit(1);
         }
-        if orig_args[1] == "rustc" {
+        if orig_args[1].ends_with("rustc") || orig_args[1].ends_with("rustc.exe") {
```

Comment:
you could convert it to a `Path` and check the `file_stem` for equality to `"rustc"`

Reply:
This is a great idea. I'll submit a new PR.